utils/markdown.go

```diff
@@ -36,10 +36,17 @@ func StripMarkdown(markdown string) (string, error) {
 }
 
 // MarkdownToHTML takes a string containing Markdown and returns a string with HTML tagged version
-func MarkdownToHTML(markdown string) (string, error) {
+func MarkdownToHTML(markdown, siteURL string) (string, error) {
+	// Turn relative links into absolute links
+	relLinkRe := regexp.MustCompile(`\[(.*)]\((/.*)\)`)
```

Comment:
Let's pre-compile this regex outside. Compiling regexes every time has a performance cost: https://developers.mattermost.com/contribute/more-info/server/style-guide/#performance-sensitive-areas